app/feature/feature-profile/src/main/kotlin/com/hedvig/android/feature/profile/tab/ProfileViewModel.kt

```diff
@@ -66,5 +66,5 @@ internal data class ProfileUiState(
   val euroBonus: EuroBonus? = null,
   val showPaymentScreen: Boolean = false,
   val memberReminders: MemberReminders = MemberReminders(),
-  val isLoading: Boolean = true,
+  val isLoading: Boolean = false,
```

Comment:
This feels like a quite significant change, the initial state now has loading as false when you first open the screen. Is this expected? I don't think we set this to true ever either from looking inside this class.

Reply:
Ill change it back!